Validation/RecoTrack/python/plotting/trackingPlots.py

```diff
@@ -214,8 +214,9 @@ def _makeMVAPlots(num, hp=False):
 )
 _effandfakeDeltaRPU = PlotGroup("effandfakeDeltaRPU",
                                 _makeEffFakeDupPlots("dr"     , "#DeltaR", effopts=dict(xtitle="TP min #DeltaR"), fakeopts=dict(xtitle="track min #DeltaR"), common=dict(xlog=True)) +
+                                _makeEffFakeDupPlots("drj" , "#DeltaR(track, jet)", effopts=dict(xtitle="#DeltaR(TP, jet)"), fakeopts=dict(xtitle="#DeltaR(track, jet)"), common=dict(xlog=True))+
```

Comment:
Either
```suggestion
                                _makeEffFakeDupPlots("drj" , "#DeltaR", effopts=dict(xtitle="#DeltaR(TP, jet)"), fakeopts=dict(xtitle="#DeltaR(track, jet)"), common=dict(xlog=True))+
```
(to use only `#DeltaR` in y axis title) or
```suggestion
                                _makeEffFakeDupPlots("drj" , "#DeltaR(track, jet)", effopts=dict(xtitle="#DeltaR(TP, jet)", ytitle="efficiency vs #DeltaR(TP, jet"), fakeopts=dict(xtitle="#DeltaR(track, jet)"), common=dict(xlog=True))+
```